src/standalone/Vegetation/Canopy.jl

```diff
@@ -510,11 +520,11 @@ function ClimaLSM.make_update_aux(
 
         # We update the fa[n_stem+n_leaf] element once we have computed transpiration, below
         # update photosynthesis and conductance terms
-        medlyn_factor .= medlyn_term.(g1, T, P, q, Ref(thermo_params))
+        medlyn_factor .= medlyn_term.(g1, T_air, P, q, Ref(thermo_params))
```

Comment:
should this be T_air? note that we pass P_air (P) and q_air (q). (perhaps we should make that clearer?)